datasets.py

```diff
@@ -162,14 +168,19 @@ def from_path(self, file_path):
         image = Image.open(file_path).convert("RGB")
         image = self.transform(image)
 
+        #Note: bbox digits should be: x, y, width, height
         person_bbox = None # optional
         gender = None # optional, we have used 0 for male and 1 for female when these labels exist (yes, this order is reversed from self.num_gender_images above)
         gender_info = [gender, person_bbox] # optional
 
         country = None # optional
 
+        #Note: This is a map of present labels: image_anns = [{‘label’: ‘mustache’}, {‘label’: ‘young’}] where the image has a 1 for mustache and young (it's present)
```

Comment:
can you change the example labels from mustache, young -> tennis ball, dog? objects may be more easily understandable use-case